### PR TITLE
Package hpke.0.0.pre-alpha

### PR DESCRIPTION
**Pull request by FantomeBeignet**
### `hpke.0.0.pre-alpha`
HPKE implementation in OCaml
RFC 9180 HPKE implementation in OCaml

---
* Homepage: https://github.com/FantomeBeignet/ohpke
* Source repo: git+https://github.com/FantomeBeignet/ohpke.git
* Bug tracker: https://github.com/FantomeBeignet/ohpke/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/hpke/hpke.0.0.pre-alpha/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+authors: ["Tom Béné <himself@fantomebeign.net>"]
+license: "ISC"
+tags: ["cryptography" "hybrid encryption" "public key"]
+homepage: "https://github.com/FantomeBeignet/ohpke"
+doc: "https://www.fantomebeig.net/projects/ohpke/docs/"
+bug-reports: "https://github.com/FantomeBeignet/ohpke/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.7"}
+  "cstruct"
```
Comment (avsm) [inline packages/hpke/hpke.0.0.pre-alpha/opam:15]: You need a minimum version bound on Cstruct here (see the various failures in CI).

**Comment by avsm** [comment]
I think the version you're looking for is probably `0.0~alpha0` or similar.  The `~` is "less than the version prefix".

**Comment by kit-ty-kate** [comment]
ping @FantomeBeignet have you had a chance to look at the review above?

**Comment by samoht** [comment]
@FantomeBeignet ping - could you update your patch to consider the review above? Or do you need help to do so?

**Comment by shonfeder** [comment]
I'm going to close this, since it has been more than one year. @FantomeBeignet please feel free to reopen whenever you have the bandwidth to see the publication thru! We appreciate your efforts to contribute to the repo :)

**Pull request closed without merging**